README.md

```diff
@@ -0,0 +1,38 @@
+[![GitHub issues](https://img.shields.io/github/issues/italia/docs.italia.it.svg)](https://github.com/italia/docs.italia.it/issues)
+[![Join the #docs-italia channel](https://img.shields.io/badge/slack%20channel-docs--italia-blue.svg)](https://developersitalia.slack.com/messages/C9T4ELD4G/)
+[![Get invited](https://slack.developers.italia.it/badge.svg)](https://slack.developers.italia.it/)
+
+# Docs Italia
+
+Docs Italia è un servizio a disposizione delle Pubbliche Amministrazioni per pubblicare documenti tecnici e amministrativi, e offre ai cittadini la possibilità di leggere e commentare documenti pubblici.
+
+Questo repository ospita la piattaforma di pubblicazione ed è un fork del progetto open-source [ReadTheDocs](http://readthedocs.org/), di cui modifica a fondo le modalità di pubblicazione e l'esperienza utente.
+
+## Indice
+
+- [Come iniziare](#come-iniziare)
+- [Segnalazione bug](#segnalazione-bug)
+- [Repository GitHub](#repository-github)
+
+## Come iniziare
+
+Se sei interessato alla pubblicazione su Docs Italia, ti rimandiamo alla Guida alla Pubblicazione; se invece vuoi contribuire al progetto, dai un'occhiata alle [indicazioni su come contribuire a Docs Italia](https://github.com/italia/docs.italia.it/blob/italia-2.3.13/CONTRIBUTING.md) e inizia installando la piattaforma con gli [scripts di deploy](https://github.com/italia/docs-italia-deploy-scripts). Alcune delle tematiche principali sono state identificate e descritte nelle [Wiki di progetto](https://github.com/italia/docs.italia.it/wiki).
+
+## Repository GitHub
+
+Il progetto Docs Italia si compone inoltre di altre componenti e guide che lo rendono un servizio completo a disposizione delle Pubbliche Amministrazioni:
+
+* Gli scripts di deploy della piattaforma: https://github.com/italia/docs-italia-deploy-scripts
+* Il tema Sphinx per i documenti: https://github.com/italia/docs-italia-theme (il nuovo tema è attualmente sul branch `bootstrap-italia`)
+* Un convertitore per documenti nei formati più diffusi: https://github.com/italia/docs-italia-comandi-conversione
+* Un'applicazione Django basata sul convertitore sopra indicato: https://github.com/italia/docs-italia-convertitore-web
+* La guida per la pubblicazione su Docs Italia: https://github.com/italia/docs-italia-guide (pubblicata [qui](http://guida-docs-italia.readthedocs.io/it/latest/))
+* Uno "starter kit" per la pubblicazione: https://github.com/italia/docs-italia-starter-kit
+
+## Segnalazione bug
+
+Vuoi segnalare un bug?
+
+Prima di tutto assicurati che il problema che desideri segnalare sia relativo al Docs Italia e non alla codebase di ReadTheDocs (in tal caso puoi fare riferimento al [repository di ReadTheDocs](https://github.com/rtfd/readthedocs.org)), poi [apri una issue](https://github.com/italia/docs.italia.it/issues) descrivendo ciò che non funziona.
```

Comment:
a Docs Italia

Uhm non farei far aprire issue upstream visto che non siamo in sync con la loro ultima versione e comunque non è polite :)